test/lint/README.md

```diff
@@ -1,5 +1,23 @@
 This folder contains lint scripts.
 
+Running locally
+===============
+
+To run linters locally with the same versions as the CI environment, use the included
+Dockerfile:
+
+```sh
+cd ./ci/lint
+docker build -t bitcoin-linter .
+
+cd /root/of/bitcoin/repo
```

Comment:
since we're already assuming starting in the root 2 lines up, I'd just replace this with

```suggestion
cd ../..
```